scripts/deploy/deploy.js

```diff
@@ -177,13 +177,16 @@ async function main() {
   const assessment = await deployProxy('Assessment', [tk.address]);
 
   console.log('Deploying SwapOperator');
-  const cowVaultRelayer = await deployImmutable('CSMockVaultRelayer');
-  const cowSettlement = await deployImmutable('CSMockSettlement', [cowVaultRelayer.address]);
+  const cowVaultRelayer = await deployImmutable('SOMockVaultRelayer');
+  const cowSettlement = await deployImmutable('SOMockSettlement', [cowVaultRelayer.address]);
   const swapOperator = await deployImmutable('SwapOperator', [
     cowSettlement.address,
     owner,
     master.address,
     '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
+    AddressZero,
```

Comment:
`0xEeee...eE` parameter is supposed to be `WETH` address now. Would you please replace `0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE` with `AddressZero` so we can deal with it later?

Reply:
Sure 👍